src/Resources/ResourceManager/Utilities/BicepUtility.cs

```diff
@@ -66,6 +66,10 @@ public static string BuildFile<T>(ScriptExecutor<T> executeScript, string bicepT
                 {
                     File.Copy(bicepTemplateFilePath, tempPath, true);
                 }
+                else
+                {
+                    throw new AzPSArgumentException(Properties.Resources.InvalidBicepFilePathOrUri, bicepTemplateFilePath);
```

Comment:
The second parameter of `AzPSArgumentException()` should be the name (instead of value) of the cmdlet parameter.